fake.txt

```diff
@@ -0,0 +1,14820 @@
+    "image": "https://source.unsplash.com/400x400/?handbags",
+    "title": "Practical Rubber Chicken",
+    "subtitle": "Accusantium dolores sapiente consectetur laboriosam molestias odit voluptatem autem odio.",
+    "description": "Sit voluptatibus et ea quisquam dolores voluptatem nihil. Nihil est omnis molestiae sequi aut et facilis voluptatem. Eos error sapiente necessitatibus doloribus cumque. Vero et id. Non aut qui aut esse.",
+    "categoryId": "fcf6139e-5d8f-4a6d-bb21-7961c0db320a",
+    "createdAt": 1710757142608,
+    "reviews": [
+      {
+        "userId": "d42940d4-6fa1-4533-90a0-9b8c40ebbead",
+        "productId": "ea83f1ff-d538-4cf9-ab6d-0a5f279cea49",
+        "rating": 3,
+        "review": "Ipsam harum quo at consequatur omnis neque id. Mollitia impedit ipsum ut quod inventore sapiente ducimus illo inventore. Perferendis sed nihil quis."
+      }
+    ],
+    "price": "833.00",
+    "brandId": "ac27c37e-3047-4ce0-a36f-db20b6ca8e8d",
+    "quantity": 60,
+    "discountPercentage": 41
+  }
+]
+
+Users:
+[
+  {
+    "id": "e2b7329d-a35f-4545-8540-12717882f510",
+    "user": "Bennie.Deckow47",
+    "password": "hECkf0tUF_ePJSl",
+    "avatar": "https://loremflickr.com/320/240/person?random=409",
+    "firstName": "Wilburn",
+    "lastName": "Renner",
+    "mobile": "873-873-3489",
+    "email": "Kirk_Cummerata27@gmail.com",
+    "address": {
+      "street": "Allie Run",
+      "city": "North Zackary",
+      "state": "North Dakota",
+      "country": "Suriname",
+      "zipcode": "80259"
+    },
+    "reviews": [
+      {
+        "userId": "e2b7329d-a35f-4545-8540-12717882f510",
+        "productId": "d929d892-950d-4339-8477-65028d34557a",
+        "rating": 1,
+        "review": "Possimus quis aut consequatur nemo maxime possimus ullam veniam. Alias velit aspernatur explicabo harum omnis."
+      }
+    ],
+    "orders": [
+      {
+        "date": 1713034325809,
+        "category": "processing",
+        "status": "unpaid",
+        "address": {
+          "street": "Abbott Stravenue",
+          "city": "Port Cooper",
+          "state": "Minnesota",
+          "country": "Republic of Korea",
+          "zipcode": "38076-3794"
+        },
+        "products": [
+          {
+            "productId": "a023d442-3cc4-41de-911d-2122bc8d04f5",
+            "quantity": 5
+          },
+          {
+            "productId": "d929d892-950d-4339-8477-65028d34557a",
+            "quantity": 6
+          }
+        ]
+      },
+      {
+        "date": 1713080536454,
+        "category": "cancelled",
+        "status": "paid",
+        "address": {
+          "street": "Corrine Crossing",
+          "city": "Davionside",
+          "state": "Michigan",
+          "country": "Kuwait",
+          "zipcode": "25314"
+        },
+        "products": [
+          {
+            "productId": "2df5de6d-65a9-4cc2-8785-0586a3504730",
+            "quantity": 6
+          }
+        ]
+      }
+    ]
```

Comment:
this should also be an object of objects with orderId as the key like this:
`{[orderId]:{
		products:[{
			productId:string,
			quantity:number,
		}],
		date:number => Date.now(),
		category:string => "cancelled" | "completed" | "processing",
		status: string => "paid" | "unpaid",
		address: Address
	}}`